### PR TITLE
Move SDK test stuff to matter test program folder

### PR DESCRIPTION
**Pull request by hiltonlima**
Created matter test program folder: `test_collection/matter`  

Moved:
`test_collections/sdk_tests` -> `test_collections/matter/sdk_tests`  
image

Some references updated

Python test PR: https://github.com/project-chip/matter-test-scripts/pull/118

### CODE REVIEW

**Review by rquidute — COMMENTED** [review]
`test_collections/sdk_tests/sdk_checkout/python_testing/scripts/custom/.gitkeep`, `test_collections/sdk_tests/sdk_checkout/yaml_tests/yaml/custom/.gitkeep` and 
`.version_information` files were deleted and no moved to new folders. 
Did you mean that?

File: test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh [hunk 1]
```diff
@@ -37,9 +37,12 @@ SDK_DATA_MODEL_PATH="src/app/zap-templates/zcl/data-model/chip"
 
 ENV_FILE="$ROOT_DIR/.env"
 
-TEST_COLLECTIONS_PATH="$ROOT_DIR/test_collections"
+TEST_COLLECTIONS_PATH="$ROOT_DIR/matter"
 TEST_COLLECTIONS_SDK_CHECKOUT_PATH="$TEST_COLLECTIONS_PATH/sdk_tests/sdk_checkout"
 
+echo "HILTON"
+echo ${TEST_COLLECTIONS_SDK_CHECKOUT_PATH}
+echo "HILTON"
```
Comment (antonio-amjr) [inline test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh:45]: Are we keeping those "Hiltons"?

Comment (hiltonlima) [inline test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh:45]:
```suggestion
```

Comment (hiltonlima) [reply test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh]: Removed

**Review by antonio-amjr — APPROVED** [review]

File: test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ SDK_DATA_MODEL_PATH="src/app/zap-templates/zcl/data-model/chip"
 
 ENV_FILE="$ROOT_DIR/.env"
 
-TEST_COLLECTIONS_PATH="$ROOT_DIR/test_collections"
+TEST_COLLECTIONS_PATH="$ROOT_DIR/matter"
```
Comment (ccruzagralopes) [inline test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh:40]: I believe that the ROOT_DIR variable should have one more "/.." and this variable be "MATTER_TEST_COLLECTIONS_PATH =$ROOT_DIR/test_collections/matter"

File: test_collections/matter/sdk_tests/sdk_checkout/python_testing/scripts/custom/.gitkeep [hunk 3]
```diff
@@ -1,3 +1,3 @@
-- This is a placeholder file for the custom Python test folder
-- Use this directory to include all customized Python test files
-- The tests will be presented in a new test collection on the TH UI
\ No newline at end of file
+- This is a placeholder file for the custom Python test folder
+- Use this directory to include all customized Python test files
+- The tests will be presented in a new test collection on the TH UI
```
Comment (hiltonlima) [inline test_collections/matter/sdk_tests/sdk_checkout/python_testing/scripts/custom/.gitkeep:3]:
```suggestion
- This is a placeholder file for the custom Python test folder
- Use this directory to include all customized Python test files
- The tests will be presented in a new test collection on the TH UI

```

File: test_collections/matter/sdk_tests/sdk_checkout/python_testing/scripts/custom/.gitkeep [hunk 4]
```diff
@@ -1,3 +1,4 @@
-- This is a placeholder file for the custom Python test folder
-- Use this directory to include all customized Python test files
-- The tests will be presented in a new test collection on the TH UI
\ No newline at end of file
+- This is a placeholder file for the custom Python test folder
+- Use this directory to include all customized Python test files
+- The tests will be presented in a new test collection on the TH UI
+
```
Comment (hiltonlima) [inline test_collections/matter/sdk_tests/sdk_checkout/python_testing/scripts/custom/.gitkeep:4]:
```suggestion
- This is a placeholder file for the custom Python test folder
- Use this directory to include all customized Python test files
- The tests will be presented in a new test collection on the TH UI
```

**Review by rquidute — APPROVED** [review]

File: test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh (hunk 2 above)
Comment (hiltonlima) [reply test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh]: Done 

TEST_COLLECTIONS_PATH removed
ROOT_DIR is now **.../test_collection/matter**

**Review by ccruzagralopes — APPROVED** [review]

**Pull request merged**